crates/holochain/src/core/sys_validate/error.rs

```diff
@@ -59,7 +59,7 @@ impl From<CounterSigningError> for SysValidationError {
     }
 }
 
-#[deprecated = "This will be replaced with SysValidationOutcome as we shouldn't treat outcomes as errors"]
+// #[deprecated = "This will be replaced with SysValidationOutcome as we shouldn't treat outcomes as errors"]
```

Comment:
not sure what this deprecation was about, but we use this type in tons of places now so it doesn't seem likely to be going away any time soon